src/ui.navigation-pagination/index.vue

```diff
@@ -9,8 +9,8 @@
         v-bind="navigationButtonAttrs"
         @click="goToFirstPage"
       >
-        <!-- @slot Use it to add something instead of the first label. -->
-        <slot name="first-label">
+        <!-- @slot Use it to add something instead of the "first" label. -->
+        <slot name="{first}">
```

Comment:
not here :) I meant in description above only

`<slot name="first">` is correct